db/firebase/firebaseUtil.js

```diff
@@ -257,27 +257,38 @@ module.exports = {
      */
     async verify(email, id, guildId) {
         let emailLowerCase = email.trim().toLowerCase();
-        let userRef = getFactotumDoc().collection('guilds').doc(guildId).collection('members').where('email', '==', emailLowerCase).limit(1);
+        let userRef = getFactotumDoc().collection('InitBotInfo').doc(guildId).collection('applicants').where('email', '==', emailLowerCase).limit(1);
         let user = (await userRef.get()).docs[0];
+        let otherRolesRef = getFactotumDoc().collection('InitBotInfo').doc(guildId).collection('otherRoles').where('email', '==', emailLowerCase).limit(1);
+        let otherRolesUser = (await otherRolesRef.get()).docs[0];
         if (user) {
             let returnTypes = [];
 
             /** @type {FirebaseUser} */
             let data = user.data();
+            if (!data?.isVerified) {
+                data.isVerified = true;
+                data.VerifiedTimestamp = admin.firestore.Timestamp.now();
```

Comment:
Is there a reason for the inconsistent casing?

Reply:
ooo i put this check so that if someone's already been verified then it wouldnt reverify them + update the verifiedTimestamp, is that what you're referring to? @daniel-panhead